hazelcast/src/main/java/com/hazelcast/collection/impl/queue/QueueContainer.java

```diff
@@ -1108,12 +1108,15 @@ private void age(QueueItem item, long currentTime) {
         totalAgedCount++;
         totalAge += elapsed;
 
+        if (minAge == -1) {
```

Comment:
This would also go away if we used `Long.MAX_VALUE` and `Long.MIN_VALUE` respectively.

Reply:
see my answer [here](https://github.com/hazelcast/hazelcast/pull/19625/files#r717362510).